PowerEditor/src/WinControls/AboutDlg/AboutDlg.cpp

```diff
@@ -235,9 +235,31 @@ intptr_t CALLBACK DebugInfoDlg::run_dlgProc(UINT message, WPARAM wParam, LPARAM
 			_debugInfoStr += nppGui._keepSessionAbsentFileEntries ? L"ON" : L"OFF";
 			_debugInfoStr += L"\r\n";
 
-			// DirectWrite
-			_debugInfoStr += L"DirectWrite : ";
-			_debugInfoStr += (nppGui._writeTechnologyEngine == directWriteTechnology) ? L"ON" : L"OFF";
+			// SC_TECHNOLOGY
+			_debugInfoStr += L"Scintilla Rendering Mode : ";
+			switch (nppGui._writeTechnologyEngine)
+			{
+				case defaultTechnology:
+					_debugInfoStr += L"SC_TECHNOLOGY_DEFAULT (0)";
+					break;
+				case directWriteTechnology:
+					_debugInfoStr += L"SC_TECHNOLOGY_DIRECTWRITE (1)";
+					break;
+				case directWriteRetainTechnology:
+					_debugInfoStr += L"SC_TECHNOLOGY_DIRECTWRITERETAIN (2)";
+					break;
+				case directWriteDcTechnology:
+					_debugInfoStr += L"SC_TECHNOLOGY_DIRECTWRITEDC (3)";
+					break;
+				case directWriteDX11Technology:
+					_debugInfoStr += L"SC_TECHNOLOGY_DIRECT_WRITE_1 (4)";

```

Comment:
Is it rather `SC_TECHNOLOGY_DIRECT_WRITE_11`?

Reply:
No. It is really marked as [SC_TECHNOLOGY_DIRECT_WRITE_1](http://hg.code.sf.net/p/scintilla/code/rev/754f7d05d59e).